src/retransmit_stage.rs

```diff
@@ -34,7 +35,7 @@ fn compute_retransmit_peers(
     fanout: usize,
     hood_size: usize,
     grow: bool,
-) -> Vec<NodeInfo> {
+) -> (Vec<NodeInfo>, Vec<NodeInfo>) {
```

Comment:
Both return values are the same type. Can you add some docs to explain which is which?